packages/cashier/src/pages/account-transfer/account-transfer-form/account-transfer-form-side-note.tsx

```diff
@@ -61,8 +62,8 @@ const AccountTransferNote = ({
                 />
             ) : (
                 <Localize
-                    i18n_default_text='You’ll not be charged a transfer fee for transfers in the same currency between your Deriv fiat and {{platform_name_mt5}} accounts.'
-                    values={{ platform_name_mt5 }}
+                    i18n_default_text='You’ll not be charged a transfer fee for transfers in the same currency between your Deriv fiat and {{platform_name_mt5}} accounts, and between your Deriv fiat and {{platform_name_ctrader}} accounts.'
+                    values={{ platform_name_mt5, platform_name_ctrader }}
                 />
```

Comment:
IMHO here we also should mention `ctrader` account because no matter is `dxtrade` transfer allowed for the user or not we should mention that the user can transfer to ctrader account.